pkg/controller/resourceconfig/static_config.go

```diff
@@ -262,6 +262,7 @@ var ControllerConfigStatic = &ResourcesControllerMap{
 	{Group: "firebasestorage.cnrm.cloud.google.com", Kind: "FirebaseStorageBucket"}:                             {DefaultController: k8s.ReconcilerTypeTerraform, SupportedControllers: []k8s.ReconcilerType{k8s.ReconcilerTypeTerraform}},
 	{Group: "firestore.cnrm.cloud.google.com", Kind: "FirestoreDatabase"}:                                       {DefaultController: k8s.ReconcilerTypeDirect, SupportedControllers: []k8s.ReconcilerType{k8s.ReconcilerTypeDirect}},
 	{Group: "firestore.cnrm.cloud.google.com", Kind: "FirestoreDocument"}:                                       {DefaultController: k8s.ReconcilerTypeDirect, SupportedControllers: []k8s.ReconcilerType{k8s.ReconcilerTypeDirect}},
+	{Group: "firestore.cnrm.cloud.google.com", Kind: "FirestoreField"}:                                          {DefaultController: k8s.ReconcilerTypeDirect, SupportedControllers: []k8s.ReconcilerType{k8s.ReconcilerTypeDirect}},
```

Comment:
Nit: It would be nice to do something like add a comment indicating alpha controllers.